MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -169,13 +169,15 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
-    func updateStreetNameForStep() {
+    func updateStreetNameForStep(currentLeg: RouteLeg? = nil) {
```

Comment:
There aren’t that many call sites, so remove the `= nil` to make it explicit when `nil` is passed in.